_docs/resources/index.md

```diff
@@ -1,7 +1,120 @@
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div ><img src="{{site.baseurl}}/assets/images/lo-fi.svg" alt="low-fi icon"/>
+  	<h4>Low-fi Stickersheet</h4>
+  </div>
+    <p>The low-fi stickersheet is a downloadable sketch file. Once downloaded, a designer can copy and paste elements from this file into your own Sketch documents to quickly prototype designs for stakeholders and designers. </p>
+    <a id="link" href="{{site.cdn_url}}/sketch/low-fi_helix_stickersheet_v0.1.sketch">Download Low-fi Sticker Sheet</a>
+</div>
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+## Online Tools
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/palette-contrast.svg" alt="color palette icon"/>
+    <h4>Palette contrast checker</h4>
+  </div>
+    <p >When finalizing color schemes for a component or layout, accessibility is always a concern. This lightweight browser app lets you select a color from the Helix palette and check readability against different backgrounds.</p>
+    <a id="link" target="_blank" href="http://citguy.com/PaletteContrast">View the tool</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/codepen.svg" alt="codepen logo icon"/>
+    <h4>Codepen Template</h4>
+  </div>
+    <p >If you are looking to test out HelixUI web components, look no further than this pen. The Helix team has created a codepen template that requires all the necesary polyfills and HelixUI code for quick and easy testing.</p>
+    <a id="link" target="_blank" href="https://codepen.io/pen?template=YaRqQg">Create a pen</a>
+</div>
+
+{% endcolumn %}
+
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+
+## GitHub Repos
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="GitHub logo icon"/>
+    <h4>Design system repo</h4>
+  </div>
+    <p>The repository is used to generate the Helix documentation site (this site). Feel free to star, submit issues, or help. Note: You will need access to the rackerlabs organization on public GitHub to view.</p>
+    <a id="link" target="_blank" href="https://github.com/rackerlabs/design-system">View the repo</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="GitHub logo icon"/>
+    <h4>HelixUI repo</h4>
```

Comment:
I'd tend to convert the dash in the `helix-ui` to a space, rather than bunching the letters together.  This would be `Helix UI repo`.

Reply:
The name is technically HelixUI and thats how it is mentioned on their site, aside from the URL. Confusing I know. Laura suggested I switch from Helix UI to HelixUI earlier in the review. I'm going to opt to leave it as is for the time being.